src/components/userProfile/components/UserProfileModal.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { Modal, Form, Input, Row, Col, Tooltip, Icon } from 'antd';
+
+import { USER_PAGE } from '../constants';
+
+const {
+    CLASSNAMES: { HEADER_INFO, INFO_DIV, ROW_CONTAINER, TITLE },
+    STRINGS: { ADDRESS, EMAIL, CITY, COUNTRY, FIRST_NAME, LAST_NAME, PHONE, USERNAME },
+    TEXTS: { BASIC_INFO_TEXT, CONTACT_INFO_TEXT, LOCATION_INFO_TEXT },
+} = USER_PAGE;
+
+const UserProfileModal = Form.create({ name: 'form_in_modal' })(
+    //   eslint-disable-next-line
+    class extends React.Component {
+        render() {
+            const {
+                form: { getFieldDecorator },
+                visible,
+                onCancel,
+                onCreate,
+                userData: {
+                    address,
+                    city,
+                    country,
+                    email,
+                    firstName,
+                    lastName,
+                    phoneNumber,
+                    username,
+                },
+            } = this.props;
+
+            const PROFILE_INFO = [
```

Comment:
why is this being defined here and not in the constants file?

Reply:
The value for the initialValue is used here. May not work if PROFILE_INFO is in the constant